c/anagrams_kata/anagrams_finder.h

```diff
@@ -0,0 +1,14 @@
+#ifndef ANAGRAMS_FINDER_H
+#define ANAGRAMS_FINDER_H
+
+typedef struct anagrams_finder_s
+{
+    char *wordlist;
+    void (*find)(struct anagrams_finder_s *_this, char *word);
```

Comment:
Thanks, I will try to adapt my example so that is is as equal as possible to the other languages, that's not always easy if you do it in C ;) But I always find it challenging to solve such problems with an ancient language like C and then compare the solution with other languages.